docs/source/tutorials/automated-publication-with-ci.md

```diff
@@ -0,0 +1,160 @@
+<!--
+SPDX-FileCopyrightText: 2023 German Aerospace Center (DLR), Forschungszentrum Jülich GmbH
+
+SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+<!-- 
+SPDX-FileContributor: Oliver Bertuch
+SPDX-FileContributor: Michael Meinel
+SPDX-FileContributor: Stephan Druskat
+-->
+
+# Set up automatic software publishing
+ 
+```{note}
+This tutorial works for repositories hosted on GitHub, and shows how to automatically publish to Zenodo Sandbox.
+Zenodo Sandbox is a "toy" repository that can be used to try things out.
+
+This tutorial should also work with the "real" Zenodo.
+```
+ 
+## Configure your .gitignore
+ 
+The HERMES workflow (`hermes`) uses temporary caches in `.hermes/`.
+Ignore this directory in your git repository.
+
+Add `.hermes/` to your `.gitignore` file:
+ 
+```{code-block} bash
+:caption: .gitignore
+
+.hermes/
+```
+
+## Provide additional metadata using CITATION.cff
+ 
+To provide high-quality citation metadata for your project and your publication,
+provide a `CITATION.cff` file in the [Citation File Format](https://citation-file-format.github.io/).
+
+If you don't have one yet,
+use the [cffinit](https://citation-file-format.github.io/cff-initializer-javascript/) website
+to create a `CITATION.cff` file.
+Save it to the root directory of your repository, and add it to version control.
+
+```bash
+git add CITATION.cff
+git commit -m "Add citation file"
+git push
+```
+ 
+## HERMES configuration
+ 
+The HERMES workflow is configured in a [TOML](https://toml.io) file: `hermes.toml`.
+Each step in the publication workflow has its own section.
+
+Configure HERMES to:
+
+- harvest metadata from Git and `CITATION.cff`
+- skip validation of `CITATION.cff`
+- deposit on Zenodo Sandbox (which is built on the InvenioRDM)
+- use Zenodo Sandbox as the target publication repository
+
+```{code-block} toml
+:caption: hermes.toml
+:name: hermes.toml
+
+[harvest]
+from = [ "cff", "git" ]
+
+[harvest.cff]
+validate = false
```

Comment:
Is there a good reason to switch off CFF validation here?

Reply:
Only that it was copied from showcase (which I don't think has "version" perhaps?). Happy to remove.